src/vs/workbench/contrib/debug/browser/variablesView.ts

```diff
@@ -270,7 +271,12 @@ class ScopesRenderer implements ITreeRenderer<IScope, FuzzyScore, IScopeTemplate
 	}
 
 	renderElement(element: ITreeNode<IScope, FuzzyScore>, index: number, templateData: IScopeTemplateData): void {
-		templateData.label.set(element.element.name, createMatches(element.filterData));
+		const name = element.element.name;
+		if (element.element instanceof ErrorScope) {
+			templateData.name.innerHTML = `<span class="error">${escape(name)}</span>`;
```

Comment:
@connor4312 this is not a good approach here. You should not write to the innerHTML of an element.
If you need something special like this I sugest that you introduce a new renderer and that you have your `ScopeErrorTemplate`. Or the Scope template data should handle both cases the error one and the regular one, writting to `innerHTML` feels like a hack.